docs/DeveloperGuide.md

```diff
@@ -295,16 +295,42 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is the `HealthSync` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Delete a person**
+**Use case: UC1 - Add a patient**
 
 **MSS**
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User requests to add a patient into the list.
+2.  HealthSync adds the target patient into the list
+    and displays the patient inside the updated list.
+3.  HealthSync <u>performs an auto-save (UC0A)</u>.
+
+    Use case ends.
+
+**Extensions**
+
+* 1a. The user does not specify one or more of the compulsory fields.
+
+  * 1a1. HealthSync shows an error message.
+
+    Use case ends.
+
+* 1b. The user specifies a name or IC that is not unique in the current list.
```

Comment:
Would it be better to rephrase as `The user specifies a name or IC that already exists in the current list` instead?

Reply:
This makes sense to me, your phrasing is less ambiguous to the user.